cumulus/parachains/runtimes/testing/penpal/src/xcm_config.rs

```diff
@@ -316,6 +316,12 @@ pub type TrustedReserves = (
 pub type TrustedTeleporters =
 	(AssetFromChain<LocalTeleportableToAssetHub, SystemAssetHubLocation>,);
 
+pub struct WaivedLocations;
+impl Contains<Location> for WaivedLocations {
+	fn contains(location: &Location) -> bool {
+		*location == Location::here()
+	}
+}
```

Comment:
There is already something you can use here:

```suggestion
pub type WaivedLocations = Equals<RootLocation>;
```

Also please make sure all the other system chain runtimes have `Equals<RootLocation>` part of their `WaivedLocations` so as not to break functionality.

Hint: they currently do not, for example [collectives-westend has it](https://github.com/paritytech/polkadot-sdk/blob/722b028d35cf5e29576184f1dfee613a2d41697c/cumulus/parachains/runtimes/collectives/collectives-westend/src/xcm_config.rs#L186), make sure the others do too.